src/components/PatientProfile/PatientProfilePage.tsx

```diff
@@ -473,7 +489,9 @@ const PatientProfilePage = ({
           <CompletePatientButton />
           {mode === 'edit' && (
             <>
-              <DeletePatientButton handleClick={handleDeleteClick} />
+              {isRestore ? null : (
+                <DeletePatientButton handleClick={handleDeleteClick} />
+              )}
```

Comment:
slightly cleaner option (the brackets could potentially be removed though)
```suggestion
              {!isRestore && (
                <DeletePatientButton handleClick={handleDeleteClick} />
              )}
```